docs/git/bitbucket-server.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: "docs"
+title: Bitbucket Server Integration
+class: documentation
+permalink: /git/bitbucket-server/
+published: true
+---
+You can use Probo with your instance of [Bitbucket Server](https://bitbucket.org/product/server). Just like the [Bitbucket Integration](/git/bitbucket/ "Bitbucket Integration"), Probo uses the build status pop-up to display the list of build steps as defined in your `.probo.yaml` file and their statuses. The link can be found in the Overview section of the pull request details page.
+
+Follow these steps to get started using Probo with your instance of Bitbucket Server:
+
+  1. <a href="#create-link">Create an Application Link in Bitbucket Server.</a>
+  2. <a href="#configure-auth">Configure authentication settings.</a>
+  3. <a href="#activate-repos">Activate repositories.</a>
+  4. <a href="#configure-webhooks">Configure webhooks.</a>
+
+<a name="create-link"></a>
+## Create an Application Link in Bitbucket Server.
+
+1. Go to the administration section of your Bitbucket Server installation and create an Application Link.
+  * Click the settings gear in the top right hand corner of Bitbucket's interface.
+  * Go to the Application Links page.
+  * Create a new link with the address `https://app.probo.ci`.
+  <a href="/images/bitbucket-server/bbserver-setup-1--create-app-link.png" data-lightbox="bbserver-setup-1">
+  <img src="/images/bitbucket-server/bbserver-setup-1--create-app-link.png" class="full-width">
+  </a>
+
+
+2. Specify `https://app.probo.ci` again as the Application URL.
+  <a href="/images/bitbucket-server/bbserver-setup-2--configure-app-link-url.png" data-lightbox="bbserver-setup-2">
+  <img src="/images/bitbucket-server/bbserver-setup-2--configure-app-link-url.png" alt="" class="full-width">
+  </a>
+
+3. Give your application a logical name, such as **Probo.CI**, and leave the rest of the application settings blank.
+  <a href="/images/bitbucket-server/bbserver-setup-3--configure-app-link-name.png" data-lightbox="bbserver-setup-3">
+  <img src="/images/bitbucket-server/bbserver-setup-3--configure-app-link-name.png" alt="" class="full-width">
+  </a>
+
+<a name="configure-auth"></a>
+## Configure authentication settings.
```

Comment:
The headings automatically have ids for you. You don't have to put in line 39, and you can change line 13 to `<a href="#configure-authentication-settings"> .... </a>`. Similar story for the other 3.